### DISCUSSION TITLE
Amazon connect disconnect event

### DISCUSSION THREAD

**Discussion by Nithishsubramaiyan** [aws/aws-sdk-js-v3#4310] [Q&A]
What kind of event is thrown if the customer disconnects the call.

I have checked the amazon connect event bus , as per the docs DISCONNECTED event is thrown only if the call is disconnected by the agent or flow action but need to know whether event is thrown if the customer disconnects the call within the flow.

**Comment by RanVaknin** [comment]
Hi @Nithishsubramaiyan ,

We need more context to help you with this question.  Which SDK client are you using? which docs are you referring to?
Not sure what you are trying to achieve.

Thanks,
Ran
